portal/src/examples/ButtonExample.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
```

Comment:
Is the idea that we only write the example here, and not in the packages? Or a duplication? Or create a file here that just imports the example from the package?

Reply:
Right now we write it here. Optimally, we could perhaps load it (and several examples) from the package. However, there might be different needs in the dev example and the example(s) used in documentation, so this is worth discussing.